core/src/test/java/com/linecorp/armeria/common/RequestContextWrapperTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import com.google.errorprone.annotations.MustBeClosed;
+
+import com.linecorp.armeria.common.util.SafeCloseable;
+import com.linecorp.armeria.server.ServiceRequestContext;
+
+class RequestContextWrapperTest {
+
+    static {
+        // Armeria properties use bare names.
+        Assertions.setExtractBareNamePropertyMethods(true);
```

Comment:
How about reverting back to the old value once this test is done? Otherwise we'll have to do this in the beginning of all tests for build reproducibility?

Reply:
Not sure - I figured it doesn't hurt to call it many times anyways in Armeria tests since Armeria uses bare name properties everywhere.

Reply:
Ah, yeah. Fine as long as future test cases call this when they use `usingRecursiveComparison`.